tests/e2e/specs/flinkArtifact.spec.ts

```diff
@@ -50,36 +50,42 @@ test.describe("Flink Artifacts", { tag: [Tag.CCloud, Tag.FlinkArtifacts] }, () =
     },
   ];
 
+  // Todo: add GCP, see https://github.com/confluentinc/vscode/issues/2817
+  const providersWithRegions = ["AWS/us-east-2", "AZURE/eastus"];
+
   for (const config of entrypoints) {
-    test(config.testName, async ({ page, electronApp }) => {
-      await setupTestEnvironment(config.entrypoint, page, electronApp);
-
-      const artifactsView = new FlinkDatabaseView(page);
-      await artifactsView.ensureExpanded();
-
-      const providerRegion = await artifactsView.loadArtifacts(config.entrypoint);
-
-      const uploadedArtifactName = await startUploadFlow(
-        config.entrypoint,
-        page,
-        electronApp,
-        artifactsView,
-        providerRegion,
-      );
-
-      // make sure Artifacts container is expanded before we check that it's uploaded (and then deleted)
-      await artifactsView.expandArtifactsContainer();
-
-      // make sure Artifacts container is expanded before we check that it's uploaded (and then deleted)
-      await artifactsView.expandArtifactsContainer();
-      await expect(artifactsView.artifacts.filter({ hasText: uploadedArtifactName })).toHaveCount(
-        1,
-      );
-
-      await artifactsView.deleteFlinkArtifact(uploadedArtifactName);
-      await expect(artifactsView.artifacts.filter({ hasText: uploadedArtifactName })).toHaveCount(
-        0,
-      );
+    test.describe(config.testName, () => {
+      for (const providerRegion of providersWithRegions) {
+        test(`with ${providerRegion}`, async ({ page, electronApp }) => {
+          await setupTestEnvironment(config.entrypoint, page, electronApp);
+
+          const [provider, region] = providerRegion.split("/");
```

Comment:
Minor: Would it be easier to set up `providersWithRegions` as an array of `{ provider: string; region: string }` objects instead of strings? Looks like most functions take them as separate arguments